src/pages/register-guide/index.tsx

```diff
@@ -19,27 +28,94 @@ export const RegisterGuide: React.FC<RegisterGuideProps> = (): JSX.Element => {
-          value={description}
-          onChange={(event) => setDescription(event.target.value)}
-        />
-        <button type="submit">Salvar</button>
-        <button type="reset">Fechar</button>
-      </form>
-    </>
+    <Grid container alignItems={'center'} justifyContent={'center'} role="main">
+      <Grid item md={6} sx={styles.content} component="section">
+        <Box sx={styles.header} component="header">
+          <Typography sx={styles.headerTitle} variant="h1">
+            CADASTRO DE GUIA
+          </Typography>
+        </Box>
+        <Box padding={'1rem 3rem'} component="section">
+          <Button
+            variant="contained"
+            sx={styles.buttonDigitalContent}
+            role="button"
+          >
+            Buscar conteúdo digital
+          </Button>
+          <Box
+            component="form"
+            onSubmit={handleSubmit}
+            flexDirection={'column'}
+            display={'flex'}
+          >
+            <InputLabel
+              htmlFor="titulo"
+              id="tituloLabel"
+              sx={styles.labelInput}
+            >
+              Título:
+            </InputLabel>
+            <InputBase
+              type="text"
+              id="titulo"
+              name="titulo"
+              role="input"
+              required
+              aria-labelledby="tituloLabel"
+              value={title}
+              sx={styles.input}
+              onChange={(event) => setTitle(event.target.value)}
+            />
+            <InputLabel
+              htmlFor="descricao"
+              sx={styles.labelInput}
+              id="descricaoLabel"
+            >
+              Descrição:
+            </InputLabel>
+            <InputBase
+              multiline={true}
+              minRows={5}
+              role="input"
+              id="descricao"
+              name="descricao"
+              aria-labelledby="descricaoLabel"
+              required
+              value={description}
+              sx={styles.input}
+              onChange={(event) => setDescription(event.target.value)}
+            />
+            <Grid
+              container
+              justifyContent={'space-evenly'}
+              alignItems={'center'}
+            >
+              <Grid item md={6} sx={styles.buttonWrapper}>
+                <Button
+                  sx={styles.button}
+                  variant="outlined"
+                  type="submit"
+                  role="button"
+                >
+                  Salvar
+                </Button>
+              </Grid>
+              <Grid item md={6} sx={styles.buttonWrapper}>
+                <Button
+                  sx={styles.button}
+                  variant="contained"
+                  type="reset"
+                  role="button"
+                >
+                  Fechar
+                </Button>
+              </Grid>
+            </Grid>
+          </Box>
+        </Box>
+      </Grid>
+    </Grid>
```

Comment:
Faltou os tabindex para acessibilidade

Reply:
Aqui @eduardoaraya eu tava vendo essas coisa ne
de ver onde precisava usar os tabindex e tudo mais
e tipo

não só nessa página de cadastro de guia como na home e na página de admin (que são as páginas que a gnt já tem né) é possível navegar por todas as páginas usando apenas o tab. Pelo o que eu entendi a gente só usa o `tabindex=0` quando queremos tornar algo que não é focável em algo que é. E a única coisa que eu consigo pensar que poderia se tornar focável são os textos, e esses pelo oq eu entendi não é recomendado que tenham um tabindex definido. Olha essa [resposta do stackoverflow](https://stackoverflow.com/a/59904018) falando sobre:
![image](https://user-images.githubusercontent.com/50814044/148769186-aa09d15b-8825-44a4-9c3c-f5b1cb3e0c0a.png)

então queria saber oq vc achaaa. tipo, eu sinto que no momento não precisa de tabindex, já que tudo que a gente está usando já é possível de focar com a tecla tab, como ta escrito [nesse site aqui](https://bitsofco.de/how-and-when-to-use-the-tabindex-attribute/):
![image](https://user-images.githubusercontent.com/50814044/148769425-33b93f5e-7ecb-4e61-a2f3-5c7c64786e1a.png)

e de fato né, se vc tentar navegar pelo site usando apenas o teclado vc vai conseguir. A única coisa talvez que eu senti falta foi um feedback mais claro de quais cards da home estão sendo selecionados. Os elementos do MUI funcionam que é uma beleza, é bem claro quando vc está em um deles, mas os cards do meio da home são mt difíceis de notar quando eles estão em foco ou não. 

um jeito de resolver isso seria com o seletor de css `:focus`, MAS imagino que se a gente converter tudo para link e botão do MUI talvez fique melhor. enfim, ve oq vc acha askdjkjakjs

Reply:
Sim ta certo que alguns elementos já tem o foco implementado por padrão. Porém a usabilidade do tabindex serve mais pra definir uma ordem que faça sentido pra essa navegação por tabs, e eu vejo que no formulário isso faz muita diferença. Talvez hoje o formulário não seja muito grande e não faça tanto sentido a implementação. Mas para formulários grandes tem uma certa relevancia a forma em que vc navega pelos inputs. Alguns entradas precisam ser feitas primeiro pois pode alterar outros campos do formulário. Por isso eu vejo importância em adicionar nos inputs e botões do formulário. 

@arthurvergacas

Reply:
hummm certo certo 
mas nesse caso q é um formulário pequeno que tem só dois campos e três botões, em que a ordem ja faz sentido de cara, vc acha que precisa??

Reply:
Sinceramente eu não sei kk, mas podemos deixar esse cara de lado e pensar nisso para os próximos. O ideal é sempre testar como esta o comportamento do tabindex. Outra coisa, ele pode se comportar diferente para navegadores diferentes.  Então é testar e ver se está de acordo com o esperado. Acho que a implementação já acaba com todos esses problemas. Maass respondendo a sua pergunta seria: título > conteúdo > Salvar > Fechar

Reply:
akjdkasjdkjasdj simmmm é vdd é vdd
bom então vou deixar ele de lado por enquanto no PR do issue la

e tava pesquisando aqui tbm sobre esse negócio da ordem ne, e olha oq tem no [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex):
![image](https://user-images.githubusercontent.com/50814044/148774945-e4a41fa4-7db4-4f2b-92a4-9bd1fabfd033.png)

E esse [outro site aqui](https://bitsofco.de/how-and-when-to-use-the-tabindex-attribute/) fala algo parecido:
![image](https://user-images.githubusercontent.com/50814044/148775302-ee77af30-5c62-4b01-b051-73eb61ff9d22.png)

Então pelo oq eu entendi é melhor mexer na ordem dos elementos no html do que no tab index em si. Mas n sei tbm né devem existir casos que precise. O google tem um artigo que explica alguns casos, mas todos eles são sobre usar `tabindex=-1` e `tabindex=0`. Não vi nenhum tópico q explicava casos de uso de tabindexes maiores que 0. 

De qualquer forma, a ordem no formulário já ta desse jeito que vc falou mesmo, então vou deixar do jeito que tá mesmo :))